docs/projects/version-control/config-as-code-reference.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Configuration as Code reference
+description: Details about the configuration as code feature.
+position: 20 
+---
+
+The configuration as code feature enables you to save some project-level settings as files in a git repository instead of SQL Server. The files are written in OCL (Octopus Configuration Language) format. Storing resources as files enables you to leverage version control features such as branching, pull requests, and reverting. In addition, how you deploy your code and your source code can now be saved in the same git repository. This page is a reference document of how the config-as-code feature works.
+
+## Octopus Project Level Only
+
+The config-as-code feature will store Octopus Project resources in git instead of SQL Server.
+
+### Resources NOT version controlled by config-as-code
```

Comment:
I think this section on resources that are not version-controlled might be better moved to after the ones that are?